src/Components/Search/Previews/Grids/ArtistSearch/MarketingCollections.tsx

```diff
@@ -136,13 +140,23 @@ export class MarketingCollectionsPreview extends React.Component<
 export const MarketingCollectionsPreviewFragmentContainer = createFragmentContainer(
   (props: MarketingCollectionsPreviewProps) => {
     return (
-      <Subscribe to={[SearchBarState]}>
-        {(searchState: SearchBarState) => {
+      <Responsive>
```

Comment:
Since we wind up only showing a subset of items in the preview based on screen size, we need to know that in `componentDidMount` when we're registering the preview items in the store.

So, this uses the deprecated component to allow access to know which breakpoint we're at, and thus how many items to register.